libraries/AP_Motors/AP_MotorsHeli.cpp

```diff
@@ -395,13 +395,14 @@ bool AP_MotorsHeli::parameter_check(bool display_msg) const
 }
 
 // reset_swash_servo
-void AP_MotorsHeli::reset_swash_servo(SRV_Channel *servo)
+void AP_MotorsHeli::reset_swash_servo(SRV_Channel::Aux_servo_function_t function)
 {
-    servo->set_range(1000);
+    // outputs are defined on a -500 to 500 range for swash servos
+    SRV_Channels::set_angle(function, 500);
```

Comment:
Why are we using angle here and not pwm?  This would set 500 centideg as the limit?  this doesn't translate to 500 pwm.

Reply:
It is equivalent to what was being done before. I know its strange, but I didn't want to change behavior. I think if you follow through what was previously done then you'll find the maths is the same (ie. current heli code only sets swash to 1000 possible steps, with 500 above trim and 500 below trim)